e2e/commonContext.go

```diff
@@ -190,9 +191,9 @@ var (
 		return ginkgo.Context(context, func() {
 			ginkgo.It("should get status for the component", func() {
 				cli := fmt.Sprintf("vela comp status %s", applicationName)
-				output, err := Exec(cli)
+				output, err := LongTimeExec(cli, 120*time.Second)
```

Comment:
It will need more time to show component status, so you set `LongTimeExec` for `vela comp status`?

Reply:
yes, `vela comp status` needs more time to get precise health status from HealthScope.